### DISCUSSION TITLE
No matching distribution found for ortool

### DISCUSSION THREAD

**Discussion by sbuccoliero** [google/or-tools#3050] [Build and configuration issues] [answered]
When i try to install or-tools i get this:

python3 -m pip install --upgrade --user ortool
ERROR: Could not find a version that satisfies the requirement ortool (from versions: none)
ERROR: No matching distribution found for ortool

I am using MacOS Big Sur 11.6
I have tried with python 3.6.15 as well as the built in MacOS which is currently 3.8.9

**Comment by Mizux** [comment]
is it a M1 cpu ?

**Reply by sbuccoliero** [reply]
no it is Intel
i was able to install eventually after upgrading pip to latest. thanks for your answer

**Comment by sbuccoliero** [comment, marked as answer]
no it is Intel
i was able to install eventually after upgrading pip to latest. thanks for your answer

**Reply by Mizux** [reply]
Thanks for the feedback !
